test/ui/views/not_found_view_test.dart

```diff
@@ -0,0 +1,46 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+// VIEW
+import 'package:notredame/ui/views/not_found_view.dart';
+
+// OTHER
+import '../../helpers.dart';
+
+void main() {
+  group('NotFoundView - ', () {
+    setUp(() async {
+      setupNavigationServiceMock();
+      setupAnalyticsServiceMock();
+    });
+
+    tearDown(() {});
+
+    group('UI - ', () {
+      testWidgets(
+          'has a go back to dashboard button and at least one text element',
+          (WidgetTester tester) async {
+        await tester.pumpWidget(localizedWidget(child: const NotFoundView(pageName: "/test")));
+        await tester.pumpAndSettle();
+
+        expect(find.byType(ElevatedButton), findsOneWidget);
+
+        expect(find.byType(Text), findsWidgets);
```

Comment:
That isn't enough, how many texts it should have?

You could also validate that the displayed text (on top of the button) contains "/test" in it. I suggest you use the find.text(textToFind) 😄